pkg/actions/lua.go

```diff
@@ -19,9 +20,11 @@ import (
 	"github.com/treeverse/lakefs/pkg/auth/model"
 	"github.com/treeverse/lakefs/pkg/graveler"
 	"github.com/treeverse/lakefs/pkg/logging"
+	"github.com/treeverse/lakefs/pkg/stats"
 )
 
 type LuaHook struct {
+	collector stats.Collector
```

Comment:
can you move this one to the end of the struct - just to have the embedded first.